### PR TITLE
Metronome / Marathon need to be prepared for GPU by default on DCOS 1.13

### PR DESCRIPTION
**Pull request by kensipe**
## High-level description

What features does this change enable? What bugs does this change fix?

- adding install controllable capabilities for turning on gpu support i…n marathon and metronome, along with validation

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-4824](https://jira.mesosphere.com/browse/DCOS_OSS-4824) Metronome needs to be prepared for GPU by default on DCOS 1.13.

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by kensipe** [comment]
@mesosphere-mergebot label ready for review

File: gen/calc.py [hunk 1]
```diff
@@ -200,6 +200,16 @@ def validate_mesos_container_log_sink(mesos_container_log_sink):
         "Container logs must go to 'journald', 'logrotate', or 'journald+logrotate'."
 
 
+def validate_metronome_gpu_scheduling_behavior(metronome_gpu_scheduling_behavior):
```
Comment (alenkacz) [inline gen/calc.py]: I don't see these methods being used.

Comment (kensipe) [reply gen/calc.py]: it is a good question.   I was assuming that `validate_`variable was auto trigged.  the `validate_mesos_container_log_sink` above this function either does that or isn't involved either:)    there is a `validate` entry in `entry` object as well though.   It isn't clear if registration in that is required.   https://github.com/dcos/dcos/blob/master/gen/calc.py#L1009

@orsenthil do you know?

Comment (timcharper) [reply gen/calc.py]: @kensipe it seems like they do need to be included in the 'validate' array:

https://github.com/dcos/dcos/blob/1fb126d8d904d7ca06f6eef17af886e42856e914/gen/calc.py#L1019

IE

```
/Users/tim/src/m8e/dcos/gen/calc.py

 626 | def validate_dns_forward_zones(dns_forward_zones):
 627 |     """
 628 |      "forward_zones": {"a.contoso.com": ["1.1.1.1:53", "2.2.2.2"],
/Users/tim/src/m8e/dcos/gen/calc.py

1018 | entry = {
1019 |     'validate': [
...
1027 |         validate_dns_bind_ip_blacklist,
1028 |         validate_dns_forward_zones,
1029 |         validate_zk_hosts,
```

**Pull request closed without merging**